app/views/controls/textfield.xml

```diff
@@ -1,5 +1,18 @@
 <Alloy>
     <Window title="Text Field">
-        <TextField class="textfield" onChange="textFieldValueChanged" onFocus="textFieldFocussed" onBlur="textFieldBlurred" />
+        <ScrollView id="textfieldsContainer" onClick="blurTextfield">
+            
+            <Label class="textLabel">Demonstration of multiple keyboard types</Label>
+            
+            <TextField class="textfield" id="regularTextfield" onChange="textFieldValueChanged" onFocus="textFieldFocussed" onBlur="textFieldBlurred" />
+            <TextField class="textfield" id="phoneTextfield" onChange="textFieldValueChanged" onFocus="textFieldFocussed" onBlur="textFieldBlurred" />
+            <TextField class="textfield" id="emailTextfield" onChange="textFieldValueChanged" onFocus="textFieldFocussed" onBlur="textFieldBlurred" />
+            <TextField class="textfield" id="passwordTextfield" onChange="textFieldValueChanged" onFocus="textFieldFocussed" onBlur="textFieldBlurred" />
+            
+            <Label class="textLabel" platform="ios">
+                You can drag down the keyboard when its visible and the view is scrollable. This is thanks to the keyboardDismissMode property on the wrapping ScrollView
+            </Label>
+            
+        </ScrollView>
```

Comment:
Open for discussion here, but I usually prefer not to use too many empty lines (only the ones in L#6 and L#11).

Reply:
I removed them for now... I do find it more clear though.